python/fledge/services/core/server.py

```diff
@@ -362,6 +363,9 @@ class Server:
     _user_idle_session_timeout = 15 * 60
     """ User idle session timeout (in minutes) """
 
+    _firewall_ip_addresses = {}
```

Comment:
May be better to make  Firewall.IPAddresses a singleton?

Reply:
yeah this can be moved to Firewall singleton itself, As handling of server module for cyclic import is pain throughout